src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -700,9 +690,9 @@ private static bool TryGetEncoding(string? charset, out System.Text.Encoding? en
         try
         {
             // Remove at most a single set of quotes.
-            if (charset.Length > 2 && charset[0] == '\"' && charset[^1] == '\"')
+            if (charset!.Length > 2 && charset[0] == '\"' && charset[charset.Length - 1] == '\"')
```

Comment:
Did you change this to support old platforms? If so please add comment.

Reply:
It’s the only way that it compiles for .net standard 2.0. I hardly think a comment is necessary, as it will not compile any other way. The code is easier to read this way anyway.

Reply:
ok